portfolio/src/main/webapp/index.html

```diff
@@ -86,6 +86,28 @@ <h1> 👋 Hi! I'm <span class = "accent">Maxwell</span>. </h1>
                 <input type="submit" class = "submit-button" onClick="loadComments(document.getElementById('commentLimit').value)"/>
             </form>
 
+            <form class = "active-on-login">
+                <input type = "radio" id = "English" name = "languageCode" value = "en">
+                <label for="languageCode">English</label>
+
+                <input type = "radio" id = "French" name = "languageCode" value = "fr">
+                <label for="languageCode">French</label>
+                
+                <input type = "radio" id = "Spanish" name = "languageCode" value = "es">
+                <label for="languageCode">Spanish</label>
+                
+                <input type = "radio" id = "Chinese" name = "languageCode" value = "zh">
+                <label for="languageCode">Chinese</label>
+                
+                <input type = "radio" id = "Arabic" name = "languageCode" value = "ar">
+                <label for="languageCode">Arabic</label>
+                
+                <input type = "radio" id = "Russian" name = "languageCode" value = "ru">
+                <label for="languageCode">Russian</label>
```

Comment:
The `<label for="...">` needs to use the `id` of the radio button, not the `name`.

You won't see this in the layout, because it only affects the logical association between the label and the control, not the physical presentation.
But you can see the effect, even if don't use a screen reader (where it is really needed), because it also has the effect of making the effective area of the radio button, checkbox, etc. much larger.

In other words, if you associate the label with the ID of the radio button, then clicking on, say "Russian" will also cause the the radio button to toggle. This is a huge convenience, as the user then doesn't have to hit just the small circle (radio) or box (checkbox), etc.  When I visit professional websites (like from my bank!) that don't do this, I find it really annoying that I have to carefully position the mouse over the box itself.

More details: https://www.w3schools.com/tags/tag_label.asp

In fact, with the association not related to the layout, an evil person could make a very surprising radio button interface, where clicking on the text next to the actual button would select a different button. The w3schools site will allow you to try that out. (But of course, you wouldn't do that for a production web page).

Reply:
Oh! I see, I'll make that change. Thanks!